### ISSUE TITLE
Got a new OAsisMist 600S.....none of the HomeKit buttons work except the warm mist.

### ISSUE THREAD

**Issue by Jimmy027777777** [pschroeder89/homebridge-levoit-humidifiers#76]
**Describe The Bug:**
Only warm mist button works. its powers the device on but not off. I would expect it to only control warm mist. The humidity level button does not power the device on, and it does not change target nudity level. The fan speed also doesn't work in auto or manual mode, totally non responsive. 

**To Reproduce:**
Only warm mist button is response so use any other button

**Expected behavior:**
I would expect the humidity level button the power the device on and off and change target humidity in auto mode. I would also expect more than just one button to work.

**Logs:**
[03/03/2024, 19:03:24] [Levoit Humidifiers] Changing Mode to sleep
[03/03/2024, 19:03:31] [Levoit Humidifiers] Setting Power to false
[03/03/2024, 19:03:31] [Levoit Humidifiers] Setting Mist Level to 2
[03/03/2024, 19:03:31] [Levoit Humidifiers] Setting Power to true
[03/03/2024, 19:03:32] [Levoit Humidifiers] Setting Mist Level to 4
[03/03/2024, 19:03:32] [Levoit Humidifiers] Setting Mist Level to 5
[03/03/2024, 19:03:32] [Levoit Humidifiers] Setting Mist Level to 8
[03/03/2024, 19:03:33] [Levoit Humidifiers] Setting Mist Level to 9
[03/03/2024, 19:04:08] [homebridge-levoit-humidifiers] This plugin generated a warning from the characteristic 'On': characteristic value expected boolean and received undefined. See https://homebridge.io/w/JtMGR for more info.
[03/03/2024, 19:07:00] [homebridge-levoit-humidifiers] This plugin generated a warning from the characteristic 'On': characteristic value expected boolean and received undefined. See https://homebridge.io/w/JtMGR for more info.
[03/03/2024, 19:08:02] [homebridge-levoit-humidifiers] This plugin generated a warning from the characteristic 'On': characteristic value expected boolean and received undefined. See https://homebridge.io/w/JtMGR for more info.
[03/03/2024, 19:08:13] [Levoit Humidifiers] Setting Power to true
[03/03/2024, 19:08:14] [Levoit Humidifiers] Setting Target Humidity to 40
[03/03/2024, 19:08:14] [Levoit Humidifiers] Setting Target Humidity to 40
[03/03/2024, 19:08:14] [Levoit Humidifiers] Setting Target Humidity to 40
[03/03/2024, 19:08:14] [Levoit Humidifiers] Setting Target Humidity to 42
[03/03/2024, 19:08:15] [Levoit Humidifiers] Setting Target Humidity to 47
[03/03/2024, 19:08:15] [Levoit Humidifiers] Setting Target Humidity to 48
[03/03/2024, 19:08:15] [Levoit Humidifiers] Setting Target Humidity to 49
[03/03/2024, 19:08:16] [Levoit Humidifiers] Setting Target Humidity to 50
[03/03/2024, 19:08:16] [Levoit Humidifiers] Setting Target Humidity to 51
[03/03/2024, 19:08:17] [Levoit Humidifiers] Setting Target Humidity to 46
[03/03/2024, 19:08:18] [Levoit Humidifiers] Setting Target Humidity to 43
[03/03/2024, 19:08:18] [Levoit Humidifiers] Setting Target Humidity to 42
[03/03/2024, 19:08:18] [Levoit Humidifiers] Setting Target Humidity to 44
[03/03/2024, 19:08:18] [Levoit Humidifiers] Setting Target Humidity to 47
[03/03/2024, 19:08:19] [Levoit Humidifiers] Setting Target Humidity to 48
[03/03/2024, 19:08:19] [Levoit Humidifiers] Setting Target Humidity to 48
[03/03/2024, 19:08:29] [Levoit Humidifiers] Setting Power to false
[03/03/2024, 19:08:30] [Levoit Humidifiers] Setting Mist Level to 2
[03/03/2024, 19:08:30] [Levoit Humidifiers] Setting Power to true
[03/03/2024, 19:08:30] [Levoit Humidifiers] Setting Mist Level to 4
[03/03/2024, 19:08:30] [Levoit Humidifiers] Setting Mist Level to 5
[03/03/2024, 19:08:31] [Levoit Humidifiers] Setting Mist Level to 5
[03/03/2024, 19:08:31] [Levoit Humidifiers] Setting Mist Level to 9
[03/03/2024, 19:08:35] [Levoit Humidifiers] Changing Mode to sleep
[03/03/2024, 19:08:36] [Levoit Humidifiers] Setting Display to true
[03/03/2024, 19:08:43] [homebridge-levoit-humidifiers] This plugin generated a warning from the characteristic 'On': characteristic value expected boolean and received undefined. See https://homebridge.io/w/JtMGR for more info.
[03/03/2024, 19:08:59] [Levoit Humidifiers] Changing Mode to sleep
[03/03/2024, 19:09:21] [Levoit Humidifiers] Changing Mode to humidity
[03/03/2024, 19:09:27] [Levoit Humidifiers] Changing Mode to sleep
[03/03/2024, 19:09:30] [Levoit Humidifiers] Changing Mode to humidity
[03/03/2024, 19:09:36] [Levoit Humidifiers] Setting Display to true
[03/03/2024, 19:09:39] [Levoit Humidifiers] Setting Display to false
[03/03/2024, 19:10:32] [Levoit Humidifiers] Setting Warm Level to 0
[03/03/2024, 19:10:32] [Levoit Humidifiers] Setting Warm Level to 0
[03/03/2024, 19:10:33] [Levoit Humidifiers] Setting Warm Level to 3
[03/03/2024, 19:10:34] [Levoit Humidifiers] Setting Warm Level to 1
[03/03/2024, 19:10:34] [Levoit Humidifiers] Setting Warm Level to 0
[03/03/2024, 19:11:44] [Levoit Humidifiers] Setting Power to true
[03/03/2024, 19:11:51] [Levoit Humidifiers] Setting Target Humidity to 60
[03/03/2024, 19:11:55] [Levoit Humidifiers] Setting Target Humidity to 50
[03/03/2024, 19:12:53] [homebridge-levoit-humidifiers] This plugin generated a warning from the characteristic 'On': characteristic value expected boolean and received undefined. See https://homebridge.io/w/JtMGR for more info.

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**:

**Comment by Jimmy027777777** [comment]
Thus plugin really doesn't work at all. none of the buttons do anything and all set back to off without command. The only way to turn the device on is with the warm mist button and that one works fine to turn warm off. 

The humidity level I assume is supposed to change the target humidity level but it doesn't, in fact it just turns to off for no reason. Im pretty amazed that absolutely nothing about this works, the 400S works fine.

**Comment by Jimmy027777777** [comment]
[03/03/2024, 19:11:51] [Levoit Humidifiers] Setting Target Humidity to 60
[03/03/2024, 19:11:55] [Levoit Humidifiers] Setting Target Humidity to 50
[03/03/2024, 19:12:53] [homebridge-levoit-humidifiers] This plugin generated a warning from the characteristic 'On': characteristic value expected boolean and received undefined. See https://homebridge.io/w/JtMGR for more info.

**Comment by Jimmy027777777** [comment]
If there is any more info I can provide please let me know. I have been uninstalled and reinstalled and still have basically zero functionality.

**Comment by Jimmy027777777** [comment]
Here is a link to it on amazon. Below is the model number i think (
LUH-O601S-KUS)

https://www.amazon.com/dp/B0C4GHYH2M?ref=ppx_yo2ov_dt_b_product_details&th=1

Brand  LEVOIT
Special Feature  Digital Display, Remote Control, Schedule, Aroma Diffuser,
Humidistat
Color  Black
Floor Area  430 Square Feet
Operation Mode  Ultrasonic
Product Dimensions  10.8"D x 6"W x 13.9"H
Item Weight  5.5 Pounds
Room Type  Living Room, Bedroom
Model Name  OasisMist 600S
Included Components  2 × Aroma Pad (1 Pre-Installed), 1 × User Manual, 1 ×
6-Liter Smart Warm & Cool Mist Humidifier, 1 × Cleaning Brush, 2 ×
Absorption Pad (1 Pre-Installed), 1 × Quick Start Guide, 1 × Water Filter
Sponge (1 Pre-Installed)
Voltage  120 Volts (AC)
Wattage  280 watts
Shape  Rectangular
Item Weight  5.5 pounds
Manufacturer  LEVOIT
ASIN  B0C4GHYH2M
Country of Origin  China
Item model number  LUH-O601S-KUS

**Comment by pschroeder89** [comment]
Yeah this is the same model.
I think the solution is simple, I can try to get to it this evening.

Basically, some of the 600s models use a different set of commands than others. So I just need to move the KUS model to the previous commands and it should hopefully work

**Comment by Jimmy027777777** [comment]
oh awesome, if you need me to test it out let me know.

**Comment by pschroeder89** [comment]
@Jimmy027777777 let me know if homebridge-levoit-humidifiers@1.14.0-dev1 works better for you

**Comment by Jimmy027777777** [comment]
I am not seeing that version available to download. How do I update from 1.12.4?

**Comment by Jimmy027777777** [comment]
working flawlessly, thank you!

**Comment by pschroeder89** [comment]
Published as 1.14.0
